File: code/src/test/scala/cern/colt/matrix/tlong/impl/DenseLongMatrix2DViewTest.scala
package cern.colt.matrix.tlong.impl

//remove if not needed
import scala.collection.JavaConversions._

class DenseLongMatrix2DViewTest(arg0: String) extends DenseLongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new DenseLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new DenseLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new DenseLongMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tlong/impl/DenseLongMatrix2DViewTest.java
package cernx.colt.matrix.tlong.impl;

public class DenseLongMatrix2DViewTest extends DenseLongMatrix2DTest {

    public DenseLongMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseLongMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
